_layouts/page.html

```diff
@@ -27,38 +27,55 @@
 <div class="page-content page-content--post">
 
     <aside class="main-sidebar">
+        <a href="#" class="main-sidebar__opener">
+            <span></span>
```

Comment:
Please add classes to this markup, so it will be obvious for everyone what the purpose of these elements.